lua/flatten/core.lua

```diff
@@ -178,15 +178,14 @@ M.edit_files = function(opts)
 		else
 			vim.cmd("tabedit " .. focus)
 		end
+		winnr = vim.api.nvim_get_current_win()
+		bufnr = vim.api.nvim_get_current_buf()
 	else
 		vim.api.nvim_err_writeln("Flatten: 'config.open.focus' expects a function or string, got " .. type(open))
 		return false
 	end
 
-	local ft = vim.bo.filetype
```

Comment:
This line had the assumption that the opened buffer was the current buffer at this stage.